src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,48 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+
+/**
+ * Shows the additions and deletions of tags on a person
+ */
+public class Tagging {
+
+    public enum TYPE {
+        ADD, DELETE
+    }
+
+    public Person getTaggedPerson() {
+        return taggedPerson;
+    }
+
+    public Tag getPersonTag() {
+        return personTag;
+    }
+
+    public TYPE getType() {
+        return type;
+    }
+
+    private final Person taggedPerson;
+    private final Tag personTag;
+    private final TYPE type;
+
```

Comment:
Take note of the convention for declaring variables. Codacy is helpful in noticing these.